src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,35 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        List<String> result = new ArrayList<>();
+
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(fileName));
+            StringBuilder stringBuilder = new StringBuilder();
+            String value = reader.readLine();
+            while (value != null) {
+
+                for (String word : value.split(" ")) {
+                    if (word.toLowerCase().charAt(0) == 'w') {
```

Comment:
```suggestion
                    if (word.startsWith(// make w constant and compare here)) {
```
startsWith will be better in this case, if in future we will to use combination of chars, etc